indico/modules/events/registration/client/js/form/FormItem.jsx

```diff
@@ -135,6 +171,9 @@ export default function FormItem({
   const showAsRequired = meta.alwaysRequired || isRequired;
   const inputRequired = !isManagement && showAsRequired;
   const htmlId = `input-${inputProps.fieldId}`;
+  const show =
+    (!conditionalField && conditionalValues === undefined) ||
+    conditionalValues.some(value => showIfFieldValues.includes(value));
```

Comment:
This breaks with the same example event I used for testing earlier, because `showIfFieldValues` is undefined.

This fixes it:

```suggestion
  const show =
    !conditionalField || conditionalValues?.some(value => showIfFieldValues.includes(value));
```

It also simplifies the first half of the condition, since the values will always be undefined if there's no conditional field.